api-model/src/test/java/io/enmasse/user/model/v1/UserModelTest.java

```diff
@@ -19,32 +22,35 @@
 public class UserModelTest {
     @Test
     public void testSerializeUserPassword() throws IOException {
-        User user = new User.Builder()
-                .setMetadata(new UserMetadata.Builder()
-                        .setName("myspace.user1")
-                        .setNamespace("ns1")
+        User user = new UserBuilder()
+                .withMetadata(new ObjectMetaBuilder()
+                        .withName("myspace.user1")
+                        .withNamespace("ns1")
                         .build())
-                .setSpec(new UserSpec.Builder()
-                        .setUsername("user1")
-                        .setAuthentication(new UserAuthentication.Builder()
-                                .setType(UserAuthenticationType.password)
-                                .setPassword("p4ssw0rd")
+                .withSpec(new UserSpecBuilder()
+                        .withUsername("user1")
+                        .withAuthentication(new UserAuthenticationBuilder()
+                                .withType(UserAuthenticationType.password)
+                                .withPassword("p4ssw0rd")
                                 .build())
-                        .setAuthorization(Arrays.asList(
-                                new UserAuthorization.Builder()
-                                        .setAddresses(Arrays.asList("queue1", "topic1"))
-                                        .setOperations(Arrays.asList(Operation.send, Operation.recv))
+                        .withAuthorization(Arrays.asList(
+                                new UserAuthorizationBuilder()
+                                        .withAddresses(Arrays.asList("queue1", "topic1"))
+                                        .withOperations(Arrays.asList(Operation.send, Operation.recv))
                                         .build(),
-                                new UserAuthorization.Builder()
-                                        .setAddresses(Arrays.asList("direct*"))
-                                        .setOperations(Arrays.asList(Operation.view))
+                                new UserAuthorizationBuilder()
+                                        .withAddresses(Arrays.asList("direct*"))
+                                        .withOperations(Arrays.asList(Operation.view))
                                         .build()))
                         .build())
                 .build();
 
         ObjectMapper mapper = new ObjectMapper();
         byte [] serialized = mapper.writeValueAsBytes(user);
 
+        Object foo = mapper.readValue(serialized, User.class);
+        System.out.println(foo.getClass());
+
```

Comment:
Was this meant to be left here? I don't see foo being used again

Reply:
Nope. Gone.